src/sinks/kafka/config.rs

```diff
@@ -20,12 +26,34 @@ use crate::{
 
 pub(crate) const QUEUED_MIN_MESSAGES: u64 = 100000;
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Migrator;
+
+impl EncodingConfigMigrator for Migrator {
```

Comment:
Would it be reasonable to make some "standard" migrators, and use those? I don't see anything that seems very specific to this sink here.

Reply:
Yes: https://github.com/vectordotdev/vector/pull/12136#discussion_r846324039 😁